testdata/p4_16_samples/sizeof-bmv2.p4

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2019 MNK Consulting, LLC.
+
+http://mnkcg.com
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <v1model.p4>
+
+typedef bit<48>  EthernetAddress;
+
+header Ethernet_h {
+    EthernetAddress dstAddr;
+    EthernetAddress srcAddr;
+    bit<16>         etherType;
+}
+
+header ipv4_t {
+    bit<4>  version;
+    bit<4>  ihl;
+    bit<6>  dscp;
+    bit<2>  ecn;
+    bit<16> totalLen;
+    bit<16> identification;
+    bit<1>  flag_rsvd;
+    bit<1>  flag_noFrag;
+    bit<1>  flag_more;
+    bit<13> fragOffset;
+    bit<8>  ttl;
+    bit<8>  protocol;
+    bit<16> hdrChecksum;
+    bit<32> srcAddr;
+    bit<32> dstAddr;
+}
+
+struct Parsed_packet {
+    Ethernet_h    ethernet;
+    ipv4_t        ipv4;
+}
+
+struct mystruct1 {
+    bit<4>  a;
+    bit<4>  b;
+}
+
+control DeparserI(packet_out packet,
+in Parsed_packet hdr) {
+    apply { packet.emit(hdr.ethernet); }
+}
+
+parser parserI(packet_in pkt,
+               out Parsed_packet hdr,
+               inout mystruct1 meta,
+               inout standard_metadata_t stdmeta) {
+    bit<32> len = 0;
+    state start {
+        bit<8> my_local = 1;
+        pkt.extract(hdr.ethernet);
+        len = hdr.ethernet.sizeBits() + hdr.ethernet.sizeBits();
```

Comment:
this is dead code. You should use the result for something.
Also, we claim that sizeBytes is a constant, so you should try to assign it to a `const bit<32>` value to see if it really works.

Reply:
The `len` is computed correctly as verified by a debugged message added to typeChecker.cpp.  I can certainly add code to use the result.